ext/TimeseriesSurrogatesVisualizations.jl

```diff
@@ -41,4 +41,22 @@ function TimeseriesSurrogates.surroplot(x, s;
     surroplot!(fig, x, s; cx, cs, nbins)
 end
 
-end
\ No newline at end of file
+function TimeseriesSurrogates.surrocompare(x, A, params; color = ("#7143E0", 0.9), N=1000, linewidth=3, kwargs...)
+    fig = Makie.Figure(resolution = (1080, 480), fontsize=22, kwargs...)
+
+    for (j, a) in enumerate(A)
+        for (i, p) in enumerate(params)
+            ax = Makie.Axis(fig[i,j])
+            hidedecorations!(ax)
+            ax.ylabelvisible = true
+            lines!(ax, surrogate(x, a(p))[end-N-100:end-100]; color, linewidth)
```

Comment:
Why is the end cut of f here...? `)[end-N-100:end-100]`

Reply:
The existing PartialRandomization method has issues with values at the edges of the time series (for both standardized and unstandardized time series). For example, the following figure is produced by the `surrocompare` function without trimming the last 100 samples:
![image](https://github.com/JuliaDynamics/TimeseriesSurrogates.jl/assets/42064608/376d5061-27f8-4aac-a2cf-3ce5c6c5229e)

Reply:
I've added a `transient` parameter to the `surrocompare` method to deal with this, rather than hard-coding the 100 samples into the function

Reply:
I don't understand thouh. This this a problem of the method? Like, if you simply evolved the whole system for 100 more steps, would everything be fine? The `transient` parameter does not really belong to `surrocompare`, i'd say remove it. Simply evolve the system longer if this is necessary, although it is not clear to me where the problem comes from precisely.